### PR TITLE
DBZ-3248 add log4j.properties file

### PR DESCRIPTION
**Pull request by fuxiao224**
Add log4j.properties file to debezium cassandra connector

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p  %X{dbz.connectorType}|%X{dbz.connectorName}|%X{dbz.connectorContext}  %m   [%c]%n
+# Root logger option
+log4j.rootLogger=INFO, stdout
+# Set up the default logging to be INFO level, then override specific units
+log4j.logger.io.debezium=DEBUG
+log4j.logger.io.debezium.embedded.EmbeddedEngine$EmbeddedConfig=WARN
+#log4j.logger.io.debezium.embedded.EmbeddedEngine=DEBUG
+log4j.logger.io.debezium.core=DEBUG
+log4j.logger.io.debezium.connector.db2=DEBUG
```
Comment (bingqinzhou) [inline src/main/resources/log4j.properties:13]: Can we change `db2` here to be `cassandra` please?

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p  %X{dbz.connectorType}|%X{dbz.connectorName}|%X{dbz.connectorContext}  %m   [%c]%n
+# Root logger option
+log4j.rootLogger=INFO, stdout
+# Set up the default logging to be INFO level, then override specific units
+log4j.logger.io.debezium=DEBUG
```
Comment (bingqinzhou) [inline src/main/resources/log4j.properties:9]: My bad, please change the logging level here back to `INFO`.

File: src/main/resources/log4j.properties [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} %-5p  %X{dbz.connectorType}|%X{dbz.connectorName}|%X{dbz.connectorContext}  %m   [%c]%n
+# Root logger option
+log4j.rootLogger=INFO, stdout
+# Set up the default logging to be INFO level, then override specific units
+log4j.logger.io.debezium=INFO
+log4j.logger.io.debezium.embedded.EmbeddedEngine$EmbeddedConfig=WARN
+#log4j.logger.io.debezium.embedded.EmbeddedEngine=DEBUG
+log4j.logger.io.debezium.core=DEBUG
+log4j.logger.io.debezium.connector.cassandra=DEBUG
```
[the remark this answers is not in the document]
Comment (fuxiao224) [reply src/main/resources/log4j.properties:13]: Sure! Will do!

**Comment by fuxiao224** [comment]
> One tiny comment. @bingqinzhou, could you merge it once that's addressed? Thx!
Thanks for your comment! I just resubmitted. FYI: It seems like Bingqin is having some difficulties logging into github recently. I'll followup with her tmr to see if she is able to log in now.

**Review by gunnarmorling — APPROVED** [review]
LGTM, thx. Could you only rework the commit using an email address linked to your GH account (so it's properly attributed to you) and then force-push. There shouldn't be the grey octocat next to the commit message if everything is alright. Thanks!

**Comment by fuxiao224** [comment]
@gunnarmorling Thanks for pointing that out! I didn't even realize this before. FYI @bingqinzhou has some issues logging into github recently, would you please merge it, or I'll merge it once you grant me committer access. Thanks!

**Comment by gunnarmorling** [comment]
Hey @fuxiao224, yepp, no problem, I'll merge it. But I'm still seeing the three commits with the Octocat icon. See here for a description: https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account. Could you configure your local git so it uses an e-mail linked as described there, then rebase and squash all the commits into one and force-push to this PR? Thx!

**Comment by fuxiao224** [comment]
Thanks for hints! It took me a while to figure out, but finally was able to overwrite default configs and resubmit! Please take a look again @gunnarmorling

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Applied, thx, @fuxiao224!

**Pull request merged**